### PR TITLE
Release 3.1.0-RC7

### PR DESCRIPTION
**Pull request by iloveeclipse**
See #111 for details.

### CODE REVIEW

**Comment by iloveeclipse** [comment]
@KengoTODA : I've pushed the changes, please check.

File: build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ plugins {
   id 'org.sonarqube' version '2.5'
 }
 
-version = '3.1.0-RC7'
+version = '4.0.0-SNAPSHOT'
```
Comment (KengoTODA) [inline build.gradle]: use '3.1.0-SNAPSHOT' instead. we should use '4.0.0-SNAPSHOT' after '3.1.0' release.

Comment (iloveeclipse) [reply build.gradle]: OK, but why this causes the build failure?

**Comment by iloveeclipse** [comment]
OK, the build passes now. But what a surprise that 4.0.0 is not allowed, end the gradle error was not really helpful for fixing the problem!

**Comment by iloveeclipse** [comment]
@KengoTODA : is it OK now?

**Pull request merged**